### PR TITLE
remove some trailing whitespace

### PR DESCRIPTION
**Pull request by SimonCropp**

### CODE REVIEW

**Review by SeanKilleen — COMMENTED** [review]
I'd suggest smaller PRs in the future against an issue that describes the work. 163 files is a lot, even if just glancing at them.

**Review by SeanKilleen — APPROVED** [review]
Reviewed all 163 files. Verified that it does what it says; just whitespace changes.

File: src/NUnitFramework/framework/Internal/TestProgressReporter.cs [hunk 1]
```diff
@@ -63,7 +63,7 @@ public void TestStarted(ITest test)
 
         /// <summary>
         /// Called when a test has finished. Sends a result summary to the callback.
-        /// to 
+        /// to
```
Comment (mikkelbu) [inline src/NUnitFramework/framework/Internal/TestProgressReporter.cs:66]:
```suggestion
```

File: src/NUnitFramework/framework/Internal/CultureDetector.cs [hunk 2]
```diff
@@ -76,7 +76,7 @@ public bool IsCultureSupported( CultureAttribute cultureAttribute )
             //catch( ArgumentException ex )
             //{
             //    reason = string.Format( "Invalid culture: {0}", ex.ParamName );
-            //    return false; 
+            //    return false;
```
Comment (mikkelbu) [inline src/NUnitFramework/framework/Internal/CultureDetector.cs:79]: I think we should rather remove the commented code

File: src/NUnitFramework/framework/Interfaces/TestStatus.cs [hunk 3]
```diff
@@ -13,7 +13,7 @@ public enum TestStatus
         Inconclusive,
 
         /// <summary>
-        /// The test has skipped 
+        /// The test has skipped
```
Comment (mikkelbu) [inline src/NUnitFramework/framework/Interfaces/TestStatus.cs:16]:
```suggestion
        /// The test has been skipped
```

File: src/NUnitFramework/framework/Constraints/AndConstraint.cs [hunk 4]
```diff
@@ -29,7 +29,7 @@ public class AndConstraint : BinaryConstraint
         public override string Description => Left.Description + " and " + Right.Description;
 
         /// <summary>
-        /// Apply both member constraints to an actual value, succeeding 
+        /// Apply both member constraints to an actual value, succeeding
```
Comment (mikkelbu) [inline src/NUnitFramework/framework/Constraints/AndConstraint.cs:32]:
```suggestion
        /// Apply both member constraints to an actual value,
```

**Review by mikkelbu — APPROVED** [review]
LGTM. I only had a few minor comments/suggestions

File: src/NUnitFramework/framework/Internal/CultureDetector.cs (hunk 2 above)
Comment (SimonCropp) [reply src/NUnitFramework/framework/Internal/CultureDetector.cs]: good point. done

**Pull request merged**